client/js/new-user.js

```diff
@@ -0,0 +1,56 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const form = document.querySelector("#formy-mcformface")
+    form.addEventListener("submit", userCheck)
+})
+
+const userNotFound = async () => {
+    const firstname = document.querySelector("#firstname").value 
+    console.log(firstname)
+    const lastname = document.querySelector("#lastname").value
+    console.log(lastname)
+    const age = document.querySelector("#age").value
+    console.log(age)
+    let response = await axios.post(`http://localhost:11000/users/`, {firstname, lastname, age});
```

Comment:
does this axios.get work? i think that second argument should be rather:
{
  firstname: firstname,
  lastname: lastname,
  age: age
}